app/models/problem.rb

```diff
@@ -75,6 +74,15 @@ def self.in_env(env)
     env.present? ? where(environment: env) : scoped
   end
 
+  def self.with_app_exclusions(exclude_apps)
+    app_names_to_exclude = exclude_apps && exclude_apps.split(',')
+    if app_names_to_exclude.is_a?(Array) && app_names_to_exclude.any?
```

Comment:
Suggestion for a simpler implementation if you want it:

``` ruby
app_names_to_exclude = exclude_apps && exclude_apps.split(',')
if app_names_to_exclude.present?
```

Reply:
implemented this suggestion!